src/borrow_tracker/mod.rs

```diff
@@ -260,6 +260,13 @@ impl GlobalStateInner {
         kind: MemoryKind,
         machine: &MiriMachine<'_>,
     ) -> AllocState {
+        let _span = enter_trace_span!(
+            "borrow_tracker",
+            borrow_tracker_fun = %"new_allocation",
```

Comment:
What does the `%` mean?

Reply:
Citing from [here](https://docs.rs/tracing/0.1.41/tracing/index.html#recording-fields):
- > The ? sigil is shorthand that specifies a field should be recorded using its [fmt::Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) implementation
- > The % sigil operates similarly, but indicates that the value should be recorded using its [fmt::Display](https://doc.rust-lang.org/nightly/core/fmt/trait.Display.html) implementation